### DISCUSSION TITLE
adding cayman S to boot screen

### DISCUSSION THREAD

**Discussion by vartech125** [Mr-MIBonk/M.I.B._More-Incredible-Bash#178] [Q&A]
Currenlty the only option for cayman is cayman and cayman gt4.  how do I get/add Cayman S?

**Comment by harman-f** [comment]
You mean the boot screen?

**Reply by vartech125** [reply]
Sorry Yes.  The when the radio boots I get a porsche crest followed by the Cayman logo.  Under Gems it allows me to change the diffrent models but for cayman it shows just cayman and cayman GT4.  it does ask for people to contribute their backups so they can add more options

**Reply by harman-f** [reply]
yes, your backup would help to add that model to list.

But, better not play with that setting ;)

**Reply by vartech125** [reply]
I was told that I need to get Cayman S added to Model_DB  how do I do that and what is needed?

**Comment by harman-f** [comment]
In your log file you will find the model ID. 
share that plus your car model.

**Reply by vartech125** [reply]
My car Model is Cayman S.  I see Cayman(074404), Cayman GTS(074424 and Cayman GT4(077454) in MODEL_DB.  This is what I see in the log if I am in the right spot  :\backup\logs MU0809-JTB-07406.09.1611830044.log
 
```
__  __     ___           ___  
|  \/  |   |_ _|         | _ ) 
| |\/| |    | |          | _ \ 
|_|  |_|ore|___|ncredible|___/ash - V3.5.0 "Spring Edition"
______________________________________________________________________

NOT FOR COMMERCIAL USE - IF YOU BOUGHT THIS YOU GOT RIPPED OFF

Logging_MU0809-MHI2_US_POG11_P3300-JTB-07406.09.1611830044:
HW Number: 051 - PN: 9P1035880H
Variant: FM2-P-TNSL-US-PO-MLP
EEPROM Variant: MU-P-LNS-US
Dataset Version: -----------
Model ID: 074404 - Car Model: Porsche Cayman
VIN -> CAR:  - MIB: 

 ls --> total 2919
      64 drwxrwxrwx 11 root      root          32768 Apr 15  2023 .
      64 drwxrwxrwx 11 root      root          32768 Apr 15  2023 ..
      64 drwxrwxrwx  2 root      root          32768 Apr 15  2023 .System Volume Information
       2 -rwxrwxrwx  1 root      root            986 Mar 12  2023 .git-blame-ignore-revs
       1 -rwxrwxrwx  1 root      root            399 Mar 12  2023 .gitattributes
      64 drwxrwxrwx  2 root      root          32768 Mar 12  2023 .github
       1 -rwxrwxrwx  1 root      root            132 Mar 12  2023 .gitignore
    1703 -rwxrwxrwx  1 root      root         871810 Mar 12  2023 2022_06_MHI2_Password_List_V4.0.pdf
       0 n---------  1 root      root              0 Apr 15  2023 =MIB
      81 -rwxrwxrwx  1 root      root          41255 Mar 12  2023 GEM.png
      30 -rwxrwxrwx  1 root      root          14939 Mar 12  2023 LICENSE
      64 drwxrwxrwx  4 root      root          32768 Mar 12  2023 Launcher
       1 -rwxrwxrwx  1 root      root             60 Mar 12  2023 M.I.B. - More Incredible Bash - Mibsolution Wiki.url
       0 -rwxrwxrwx  1 root      root              0 Mar 12  2023 NOT FOR COMMERCIAL USE - IF YOU BOUGHT THIS YOU GOT RIPPED OFF.txt
     284 -rwxrwxrwx  1 root      root         144962 Mar 12  2023 PATCH COMPATIBILITY TABLE.pdf
      13 -rwxrwxrwx  1 root      root           6438 Mar 12  2023 README.md
       0 -rwxrwxrwx  1 root      root              0 Mar 12  2023 Swdlautorun.txt
       1 -rwxrwxrwx  1 root      root             24 Mar 12  2023 VERSION
       1 -rwxrwxrwx  1 root      root            296 Jan 01  1980 WMPInfo.xml
      64 drwxrwxrwx  3 root      root          32768 Mar 12  2023 apps
      64 drwxrwxrwx  4 root      root          32768 Jan 01  1980 backup
      77 -rwxrwxrwx  1 root      root          39351 Mar 12  2023 bash.png
      64 drwxrwxrwx  2 root      root          32768 Mar 12  2023 config
      64 drwxrwxrwx  3 root      root          32768 Mar 12  2023 esd
       4 -rwxrwxrwx  1 root      root           1995 Mar 12  2023 metainfo2.txt
      64 drwxrwxrwx 10 root      root          32768 Mar 12  2023 mod
      64 drwxrwxrwx  4 root      root          32768 Mar 12  2023 patches
      16 -rwxrwxrwx  1 root      root           7874 Mar 12  2023 start

RCC-backup---->
ifs-root-part2 (ASCII): 00f0
3 image type=2(IFS     )  ba0000...1a8a1ff (15639040)|93...213 (120):LOCKED
0x00ba0000 - 00A2EE00

Found JTB-07406.09.1611830044 Backup, will check if all complete...
Version info..... -> done :-)
FEC folders...... -> done :-)
Shadow files..... -> done :-)
Dataset's........ -> done :-)
EEPROM........... -> done :-)
storage1......... -> done :-)
storage2......... -> done :-)
backing up backup -> part 1 done
ls -als /net/mmx/mnt/boardbook --> total 192
      64 drwxrwxrwx  3 root      root          32768 Jan 01 00:02 .
      64 drwxrwxr-x  5 root      root          32768 Jan 01 00:02 ..
      64 drwxrwxrwx  3 root      root          32768 Jan 01 00:02 MU0809
Free space on /net/mmx/mnt/boardbook 1012M


RCC-launcher---->
-- Swdlautorun.txt disabled
-- Skip MOST enabled
-- Illegal and withdrawn FECs removed
-- Cleanup done!
AppDevelopment.jar found
-- GEM version OK - no update needed
-- M.I.B GEM link installed
-- Development Mode - GEM - activated

RCC-backup---->
ifs-root-part2 (ASCII): 00f0
3 image type=2(IFS     )  ba0000...1a8a1ff (15639040)|93...213 (120):LOCKED
0x00ba0000 - 00A2EE00

Found JTB-07406.09.1611830044 Backup, will check if all complete...
Version info..... -> Backup is already there!
FEC folders...... -> Backup is already there!
Shadow files..... -> Backup is already there!
Dataset's........ -> Backup is already there!
EEPROM........... -> Backup is already there!
storage1......... -> Backup is already there!
storage2......... -> Backup is already there!
backing up backup -> Backup part 1 is already there!
ls -als /net/mmx/mnt/boardbook --> total 192
      64 drwxrwxrwx  3 root      root          32768 Jan 01 00:02 .
      64 drwxrwxr-x  5 root      root          32768 Jan 01 00:02 ..
      64 drwxrwxrwx  3 root      root          32768 Jan 01 00:02 MU0809
Free space on /net/mmx/mnt/boardbook 1012M
```

I read that car ID's for startup animation is byte 18 and that they are as follows for porsche
ID (DEC) - Model Name
0 - Macan
1 - 911
2 - 911 Carrera Coupe
3 - 911 Carrera 2S Coupe
4 - 911 Carrera 4 Coupe
5 - 911 Carrera 4S Coupe
6 - 911 Carrera Cabrio
7 - 911 Carrera 2S Cabrio
8 - 911 Carrera 4 Cabrio
9 - 911 Carrera 4S Cabrio
10 - 911 Targa 4
11 - 911 Targa 4S
12 - 911 Turbo Coupe
13 - 911 Turbo Cabrio
14 - 911 Turbo S Coupe
15 - 911 Turbo S Cabrio
16 - 911 GT3
17 - 911 GT2
18 - 911 GT3 RS
19 - 911 GT2 RS
20 - 911 Carrera Coupe
21 - Carrera Cabrio
22 - 911 Carrera gts Coupe
23 - 911 Carrera 4 gts Coupe
24 - 911 Carrera gts Cabrio
25 - 911 Carrera 4 gts Cabrio
26 - 911 Targa 4 GTS
27 - 911 Targa 4
28 - 911 Targa 4S
52 - Boxster
53 - Boxster s
54 - Boxster rs
55 - Boxster gts
56 - Boxster spyder
60 - Cayman
61 - Cayman s
62 - Cayman rs
63 - Cayman gts
64 - Cayman gt4
82 - Panamera
112 - Cayenne
113 - Cayenne s
114 - Cayenne turbo
115 - Cayenne turbo s
116 - Cayenne gts
117 - Cayenne s hybrid
118 - Cayenne diesel
120 - Cayenne diesel s
121 - Cayenne s hybrid
122 - Cayenne s e-hybrid
141 - Macan
142 - Macan s
143 - Macan hybrid
144 - Macan s ehybrid
145 - Macan turbo
146 - Macan turbo s
147 - Macan gts
148 - Macan diesel
149 - Macan diesel s

te 03 tells the car what model it is 
I am also not seeing Sport Chrono in PCM  how do fix that.  I believe its FEC 00060500 Sport Chrono

**Reply by harman-f** [reply]
Your Model ID: 074404
I'm not sure it the Model ID is really specific for the S version or if just Cayman is also "right" here.
Car Model: Porsche Cayman

"te 03" what do you mean by that?

Try to add the Fec, if not enabled yet.

**Reply by vartech125** [reply]
I saw that in a forum about decoding the pcm 4.0 for porsche 911 and he mentioned it.  The issue is the pcm and the cluster does not match which causes an error.  there are versions for different caymans the GTS the and the GT4 there should be one that would allow the PCM to show the graphics of the Cayman S just like it does for GTS and GT4 and base  Also I think the issue is since the PCM and the cluster does not match I am not seeing the Chrono on the PCM. Not sure what the PCM looks at to display the Porsche silhouette.

**Reply by harman-f** [reply]
Did you retrofit the unit?

Otherwise the model ID is factory coding on your unit and no need to change it.

**Reply by vartech125** [reply]
I did retrofit the unit I was told to convert listed ID to hex before using so ID 60 is 074404 = Cayman. what would the hex be for 61 to get cayman S.   64 is gt4
